automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimePatchSpec.scala

```diff
@@ -48,7 +48,9 @@ class RuntimePatchSpec extends FixtureAnyFreeSpec with LeonardoTestUtils with Le
                                                     ),
                                                     true,
                                                     None,
-                                                    None)
+                                                    None,
+                                                    Map("" -> ""),
+                                                    List(""))
```

Comment:
if you just need empty Map and empty List, you can do `Map.empty`, and `List.empty`